simtools/db/db_handler.py

```diff
@@ -1102,7 +1102,7 @@ def get_all_versions(
         if _cache_key not in DatabaseHandler.model_versions_cached:
             db_collection = DatabaseHandler.db_client[self._get_db_name()][collection]
             DatabaseHandler.model_versions_cached[_cache_key] = list(
-                set(post["version"] for post in db_collection.find(query))
+                {post["version"] for post in db_collection.find(query)}
```

Comment:
removal of set could lead to two identical entries here depending on the query?

Reply:
The two lines are identical. `{a for b in c}` is set comprehension (I didn't know that before ruff pointed it out).